### PR TITLE
finished summary section for Html report

### PR DESCRIPTION
**Pull request by Dragos0000**

### CODE REVIEW

File: src/common/fetchers.xsl [hunk 1]
```diff
@@ -182,5 +182,26 @@
...
+        <xd:desc>fetch all class distinct names</xd:desc>
+        <xd:param name="root"/>
+    </xd:doc>
+    <xsl:function name="f:getDistinctClassNames" as="xs:string*">
+        <xsl:param name="root" as="node()"/>
+        <xsl:sequence select="fn:distinct-values($root//element[@xmi:type = 'uml:Class']/@name)"/>
+    </xsl:function>
+    
+    <xd:doc>
+        <xd:desc>fetch class name with a given attribute name</xd:desc>
```
Comment (costezki) [inline src/common/fetchers.xsl]: do we still need this? Is it used anywhere?

File: src/html-conventions-lib/info-box.xsl [hunk 2]
```diff
@@ -0,0 +1,176 @@
...
+    xmlns:uml="http://www.omg.org/spec/UML/20131001"
+    xmlns:xmi="http://www.omg.org/spec/XMI/20131001"
+    xmlns:umldi="http://www.omg.org/spec/UML/20131001/UMLDI"
+    xmlns:dc="http://www.omg.org/spec/UML/20131001/UMLDC" xmlns:owl="http://www.w3.org/2002/07/owl#"
+    xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
+    xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dct="http://purl.org/dc/terms/"
+    xmlns:skos="http://www.w3.org/2004/02/skos/core#"
+    xmlns:f="http://https://github.com/costezki/model2owl#" version="3.0">
+
+    <xsl:import href="../common/checkers.xsl"/>
```
Comment (costezki) [inline src/html-conventions-lib/info-box.xsl]: do we also need checkers in the infobox section or just fetchers?

File: src/html-conventions-lib/info-box.xsl [hunk 3]
```diff
@@ -0,0 +1,176 @@
...
+                                <xsl:with-param name="root" select="$root"/>
+                            </xsl:call-template>
+                        </ul>
+                    </div>
+                </div>
+            </li>
+        </xsl:for-each>
+    </xsl:template>
+
+    <xsl:template name="moreDetailsForConnectors">
```
Comment (costezki) [inline src/html-conventions-lib/info-box.xsl]: more? are there also less of them? perhaps a better name would be more suggestive.

**Review by costezki — APPROVED** [review]
Well done. 
Please do the two updated before the merge.

File: src/common/fetchers.xsl (hunk 1 above)
Comment (Dragos0000) [reply src/common/fetchers.xsl]: done

File: src/html-conventions-lib/info-box.xsl (hunk 2 above)
Comment (Dragos0000) [reply src/html-conventions-lib/info-box.xsl]: yes this is not in use.

File: src/html-conventions-lib/info-box.xsl (hunk 3 above)
Comment (Dragos0000) [reply src/html-conventions-lib/info-box.xsl]: renamed

**Pull request merged**